mnefun/_report.py

```diff
@@ -299,7 +299,7 @@ def _peak_times(evoked, cov, max_peaks=5):
     times = peaks[prms.argsort()[::-1]][:npeaks]
     times.sort()
     if not len(times):  # guarantee at least 1
-        times = gfp.argmax()
+        times = [gfp.argmax()]
```

Comment:
Would tuple work as well?

Reply:
Tuples and lists are often interchangeable, but one notable time they are not is in `__getitem__` as the tuple gets parsed as a set of arguments:
```
>>> x = np.zeros(1)
>>> x
array([0.])
>>> x[0]
0.0
>>> x[(0,)]
0.0
>>> x[[0]]
array([0.])
```
So a tuple of `(0,)` gets processed the same way as `0` rather than like `[0]`.